Contrib/ChiralPairs/ChiralDescriptors.py

```diff
@@ -0,0 +1,300 @@
+#
+#  Copyright (c) 2017, Novartis Institutes for BioMedical Research Inc.
+#  All rights reserved.
+# 
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are
+# met: 
+#
+#     * Redistributions of source code must retain the above copyright 
+#       notice, this list of conditions and the following disclaimer.
+#     * Redistributions in binary form must reproduce the above
+#       copyright notice, this list of conditions and the following 
+#       disclaimer in the documentation and/or other materials provided 
+#       with the distribution.
+#     * Neither the name of Novartis Institutes for BioMedical Research Inc. 
+#       nor the names of its contributors may be used to endorse or promote 
+#       products derived from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+# A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+# OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+# SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+# LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+# THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+# Created by Nadine Schneider & Peter Ertl, July 2017
+
+
+from collections import defaultdict, Counter, namedtuple
+
+import seaborn as sns
+import numpy as np
+import re
+
+from rdkit import Chem
+from rdkit.Chem import AllChem
+from rdkit.Chem.Draw import rdMolDraw2D
+
+# build an svg grid image to print
+def _svgsToGrid(svgs, labels, svgsPerRow=4,molSize=(250,150),fontSize=12):
+    
+    matcher = re.compile(r'^(<.*>\n)(<rect .*</rect>\n)(.*)</svg>',re.DOTALL) 
+    hdr='' 
+    ftr='</svg>' 
+    rect='' 
+    nRows = len(svgs)//svgsPerRow 
+    if len(svgs)%svgsPerRow : nRows+=1 
+    blocks = ['']*(nRows*svgsPerRow)
+    labelSizeDist = fontSize*5
+    fullSize=(svgsPerRow*(molSize[0]+molSize[0]/10.0),nRows*(molSize[1]+labelSizeDist))
+    print(fullSize)
+
+    count=0
+    for svg,name in zip(svgs,labels):
+        h,r,b = matcher.match(svg).groups()
+        if hdr == '': 
+            hdr = h.replace("width='"+str(molSize[0])+"px'","width='%dpx'"%fullSize[0])
+            hdr = hdr.replace("height='"+str(molSize[1])+"px'","height='%dpx'"%fullSize[1])
```

Comment:
With String.format, this may be a bit more readable
```
"width='{}px'".format(molSize[0])
"width='"+str(molSize[0])+"px'"
```
In other cases, you use double quotes for the attributes, 
```
'width="{}px"'.format(molSize[0])
```
Or you drop support for any Python version lower than 3.6 and use f-strings
```
f'width="{molSize[0]}px"'
```